cli/cmd/uninstall.go

```diff
@@ -0,0 +1,399 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/spf13/cobra"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	apiregistrationv1client "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	// YamlSep is used to separate manifests on stdout print
+	YamlSep = "---\n"
+)
+
+type (
+	kubernetesResource struct {
+		APIVersion string             `json:"apiVersion"`
+		Kind       string             `json:"kind"`
+		Metadata   kubernetesMetadata `json:"metadata"`
+	}
+
+	kubernetesMetadata struct {
```

Comment:
can we use [ObjectMeta](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta) instead?

Reply:
The embedded fields made everything much better, amazing suggestion, thank you.